pom.xml

```diff
@@ -392,6 +392,12 @@
                 <groupId>com.github.rholder</groupId>
                 <artifactId>guava-retrying</artifactId>
                 <version>${retrying.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>com.google.guava</groupId>
+                        <artifactId>guava</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
just curious, do you know why ksql sets the guava version here https://github.com/confluentinc/ksql/blob/master/pom.xml#L118 instead of using the one from [common](https://github.com/confluentinc/common/blob/master/pom.xml#L66). might be easier to use the one from `common` to keep track of changes that arise due to CVEs etc?

Reply:
Good question.  I'm unsure.  It was quick enough to find the common properties; I've tossed up a PR to discuss removing most of the common properties.  

Notably, I have kept the netty.version since ksqlDB manages its own version of VertX and Netty is a dependency.